### PR TITLE
Add Solarc-Dark GTK Theme

### PR DESCRIPTION
**Pull request by Preisschild**
Github Repo here: https://github.com/schemar/solarc-theme

### CODE REVIEW

File: org.gtk.Gtk3theme.SolArc-Dark.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "id": "org.gtk.Gtk3theme.SolArc-Dark",
+  "branch": "3.22",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
```
Comment (bilelmoussaoui) [inline org.gtk.Gtk3theme.SolArc-Dark.json]: Any reason for not using 18.08?  I have no idea if the Gtk themes do work as expected with 18.08.

File: org.gtk.Gtk3theme.SolArc-Dark.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          "url": "https://github.com/schemar/solarc-theme/archive/v1.0.2.tar.gz",
+          "sha256": "a7384e1a00b831a82ffbc8952620103a42216e9352d8d81809ba3e34f5b69eae"
+        }
+      ]
+    },
+    {
+      "name": "appdata",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p ${FLATPAK_DEST}/share/appdata",
```
Comment (bilelmoussaoui) [inline org.gtk.Gtk3theme.SolArc-Dark.json]: Just use install here instead.

**Comment by Preisschild** [comment]
I used [org.gtk.Gtk3theme.Arc-Dark](https://github.com/flathub/org.gtk.Gtk3theme.Arc-Dark) as guide, but I can change it If you think its better.

Also I already tested it and it worked fine.

**Comment by bilelmoussaoui** [comment]
Some old repositories need to be updated, but that's a task for another day. I would wait until some of the maintainers of Flathub to answer my question about whether or not we can push Gtk themes to 18.08 freedesktop runtime or not.

**Comment by Preisschild** [comment]
Meanwhile I replaced the mkdir & cp stuff with INSTALL.

**Comment by Preisschild** [comment]
Using 18.08 returns this error: `configure: error: Invalid GNOME version: 3.24`

**Comment by Preisschild** [comment]
Nevermind, a newer version fixed this. 

It's working with 18.08 now.

**Comment by Preisschild** [comment]
bot, build org.gtk.Gtk3theme.SolArc-Dark

**Comment by Preisschild** [comment]
@bilelmoussaoui Anything new? I've updated to the latest SDK and tested it.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.gtk.Gtk3theme.SolArc-Dark

Thanks!

**Pull request closed without merging**